tests/console/yast2_apparmor.pm

```diff
@@ -24,11 +24,9 @@ sub run {
     assert_script_run("/usr/bin/zypper -n -q in yast2-apparmor");
 
     # start apparmor configuration
-    script_run("yast2 apparmor; echo yast2-apparmor-status-\$? > /dev/$serialdev", 0);
-
     # check Apparmor Configuration is opened
+    script_run("yast2 apparmor; echo yast2-apparmor-status-\$? > /dev/$serialdev", 0);
```

Comment:
that should go back to above, not here because it does not fit to the comment anymore